content/docs/command-reference/remote/add.md

```diff
@@ -299,7 +299,7 @@ $ export OSS_ACCESS_KEY_SECRET='AccessKeySecret'
 ### Click for SSH
 
 ```dvc
-$ dvc remote add -d myremote ssh://user@example.com/path/to/dir
+$ dvc remote add -d myremote ssh://user@example.com/absolute/path
```

Comment:
why is it just absolute path here if we use sftp in other places?

Reply:
Thanks. I removed these now and standardized path examples and SFTP notes in remote add/modify as well as the external data guides. See f68eb406de63c813baa2f550e302117a2da3bf0b.